### PR TITLE
fix: Add Prisma docs on how to connect to postgres

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Adds Prisma documentation on the section [Connecting To Postgres](https://supabase.com/docs/guides/database/connecting-to-postgres#integrations)

## Additional context
Missing image is due to not being ready to be shared
Screenshot 2024-01-16 at 23 15 04

### CODE REVIEW

File: apps/docs/pages/guides/database/connecting-to-postgres.mdx [hunk 1]
```diff
@@ -150,6 +150,86 @@ You can obtain your connection info and Server root certificate from your applic
 
   </StepHikeCompact.Step>
 
+</StepHikeCompact>
+### Connecting with Prisma
+
+[Prisma](https://www.prisma.io/) is a TypeScript ORM for SQL databases focused on .
```
Comment (charislam) [inline apps/docs/pages/guides/database/connecting-to-postgres.mdx:156]:
```suggestion
[Prisma](https://www.prisma.io/) is another popular TypeScript ORM for SQL databases.
```

File: apps/docs/pages/guides/database/connecting-to-postgres.mdx [hunk 2]
```diff
@@ -150,6 +150,86 @@ You can obtain your connection info and Server root certificate from your applic
...
+Currently there's a limitation where you can't use prepared statements with Prisma and Supabase. This is a known issue and we're working on the fix.
```
Comment (charislam) [inline apps/docs/pages/guides/database/connecting-to-postgres.mdx:157]:
```suggestion

<Admonition type="note">

Currently there's a limitation where you can't use prepared statements with Prisma and Supabase. This is a known issue and we're working on the fix.

</Admonition>

```

File: apps/docs/pages/guides/database/connecting-to-postgres.mdx [hunk 3]
```diff
@@ -150,6 +150,86 @@ You can obtain your connection info and Server root certificate from your applic
...
+
+<StepHikeCompact>
+  <StepHikeCompact.Step step={1}>
+
+    <StepHikeCompact.Details title="Set your connection pool to Transaction">
+
+    Change the connection pool in your project to `Transaction` by following changing it in the [Pool Mode](https://supabase.com/dashboard/project/_/settings/database) dropbox.
```
Comment (charislam) [inline apps/docs/pages/guides/database/connecting-to-postgres.mdx:164]:
```suggestion
    Make sure your connection pool mode is set to `Transaction` in your [Database Settings](https://supabase.com/dashboard/project/_/settings/database).
```

File: apps/docs/pages/guides/database/connecting-to-postgres.mdx [hunk 4]
```diff
@@ -150,6 +150,86 @@ You can obtain your connection info and Server root certificate from your applic
...
+
+    </StepHikeCompact.Details>
+    <StepHikeCompact.Code>
+
+     ![Change Connection pool to Transaction](/docs/img/guides/database/pool-mode-transaction.png)
```
Comment (charislam) [inline apps/docs/pages/guides/database/connecting-to-postgres.mdx:169]:
```suggestion
     ![Change Connection pool to Transaction](/docs/img/database/pool-mode-transaction.png)
```

**Review by charislam — APPROVED** [review]

**Review by inian — CHANGES_REQUESTED** [review]
lets update the prisma integration guide https://supabase.com/partners/integrations/prisma instead of updating this page

**Pull request closed without merging**